Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/PhysicsSetupManipulatorBus.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+
+namespace EMotionFX
+{
+    //! Provides a bus to tell the physics setup manipulators when their underlying properties have changed.
+    class PhysicsSetupManipulatorRequests
+        : public AZ::EBusTraits
+    {
+    public:
```

Comment:
suggestion: Consider adding an address for this (`ManipulatorId` maybe?) so this doesn't always have to be global

Reply:
In the place where the bus call is made it would be pretty tricky to find out which manipulator is active (and it probably doesn't make sense for it to know). And since there can't be many manipulators active, I don't see a problem with having it unaddressed.